tools/src/bin/sha256tree-benching.rs

```diff
@@ -7,6 +7,12 @@ use std::fs::File;
 use std::io::{BufWriter, Write};
 use std::time::Instant;
 
+/*
+This file is for comparing the native sha256tree with the clvm implementation which previously existed.
+The costs for the native implementation should be lower as it is not required to make allocations.
+*/
```

Comment:
It's not clear to me how the output of this program should be interpreted. We want it to tell whether the current cost for the native shatree operator is reasonable compared to the CLVM implementation.

How can we tell?

Why does the actual timing matter? It's just the cost that matters, isn't it?